source/idn-homograph/domains.csv

```diff
@@ -0,0 +1,4 @@
+ɡithub.com
```

Comment:
Punicode... only

Each record should then have a issue, describing the records, see full example in https://github.com/mypdns/matrix/issues/1300

Reply:
Oh, I got your point now. Ok, I will do it